java/connector-node/risingwave-sink-mock-flink/risingwave-sink-mock-flink-http-sink/pom.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>com.risingwave</groupId>
+        <artifactId>risingwave-sink-mock-flink</artifactId>
+        <version>0.1.0-SNAPSHOT</version>
+    </parent>
+
+    <groupId>com.risingwave.mock.flink.sink</groupId>
+    <artifactId>risingwave-sink-mock-flink-http-sink</artifactId>
+
+    <properties>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    </properties>
+    <dependencies>
+        <dependency>
+            <groupId>com.risingwave.mock.flink.sink</groupId>
+            <artifactId>risingwave-sink-mock-flink-common</artifactId>
+            <version>0.1.0-SNAPSHOT</version>
+        </dependency>
+        <!-- https://mvnrepository.com/artifact/com.getindata/flink-http-connector -->
+        <dependency>
+            <groupId>com.getindata</groupId>
+            <artifactId>flink-http-connector</artifactId>
+            <version>0.10.0</version>
+        </dependency>
+        <dependency>
```

Comment:
The `connector-base` can be moved to the `mock-runtime` module, since no code in the `mock-flink-http-sink` module is referring to any class in `connector-base`, and the code can be compiled without this dependency. We only need to ensure that the jar of `connector-base` module  is present at runtime.

Reply:
It might be a good idea, but putting Flink's dependencyManagement together may pose some problems. Different sink implementations have their own ideas, and they might depend on versions that other sinks do not. Putting them together means importing everything each time, which could undermine the modular approach we discussed earlier.

Reply:
But eventually there can be only one supported flink version running in our system, because if there are flink dependencies of multiple versions, a random one will override the other versions, so we should enforce to use the same flink version. 

Besides, do you have any reproducible concrete example for flink's sink interface having backward compatibility issue yet. I haven't see it yet, and for a mature open-source project I think they should maintain backward compatibility for these interfaces open to ecosystem developers.

Reply:
`dependencyManagement` is not `dependency`. `dependencyManagement` only manages the concrete version of each dependency used across all modules. Including a dependency in the management section does not mean all its submodules are including the dependency. The dependency is only included in the submodule when the dependency is added to its `dependency` section, and if not added, the dependency will not be included.